src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/, 'g');
```

Comment:
@aayushsanghavi looks like @Pomax has given you a good way forward here, are you willing to finish it?